py/DREAM/Output/FluidQuantity.py

```diff
@@ -152,7 +152,7 @@ def get(self, r=None, t=None):
             return self.data[t,r]
 
         
-    def plot(self, ax=None, show=None, r=None, t=None, log=False, colorbar=True, VpVol=False, weight=None, **kwargs):
+    def plot(self, ax=None, show=None, r=None, t=None, log=False, colorbar=True, VpVol=False, weight=None, ms_time=False, **kwargs):
```

Comment:
Ooh, this looks like a good idea! Maybe we could even generalize it further so that it's a string indicating if we want seconds, milliseconds, microseconds or nanoseconds? I can do that.